appveyor.yml

```diff
@@ -10,4 +10,5 @@ build_script:
   - build.cmd --quiet verify
 clone_depth: 1
 test: off
-deploy: off
\ No newline at end of file
+deploy: off
+os: Visual Studio 2017 RC
```

Comment:
Yes. The default image doesn't have .NET Framework 4.6, which is required for dotnet-test